Mlem/Views/Tabs/Settings/Components/Views/Appearance/AppearanceSettingsView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  Appearance.swift
+//  Mlem
+//
+//  Created by David Bureš on 08.05.2023.
+//
+
+import SwiftUI
+
+struct AppearanceSettingsView: View {
+
+    @AppStorage("lightOrDarkMode") var lightOrDarkMode: UIUserInterfaceStyle = .unspecified
+    
+    var body: some View {
+        VStack {
+            List {
+                Section {
+                    NavigationLink {
+                        ThemeSettingsView()
+                    } label: {
+                        switch lightOrDarkMode {
+                        case .unspecified:
+                            ThemeLabel(title: "Theme", color1: .white, color2: .black)
+                        case .light:
+                            ThemeLabel(title: "Theme", color1: .white)
+                        case .dark:
+                            ThemeLabel(title: "Theme", color1: .black)
+                        default:
+                            ThemeLabel(title: "Theme", color1: .clear)
+                        }
+                    }
+                    #if !os(macOS) && !targetEnvironment(macCatalyst)
+                    NavigationLink {
+                        IconSettingsView()
+                    } label: {
+                        Label {
+                           Text("App Icon")
+                        } icon: {
+                            IconSettingsView.getCurrentIcon()
+                                .resizable()
+                                .scaledToFit()
+                                .frame(width: 28, height: 28)
```

Comment:
This 28 value gets reused a bunch--it should live in AppConstants as `settingsIconSize` or something like that